daemon/call_interfaces.c

```diff
@@ -1730,7 +1730,7 @@ static void ng_stats_ssrc(bencode_item_t *dict, struct ssrc_hash *ht) {
 		if (!se->stats_blocks.length || !se->lowest_mos || !se->highest_mos)
 			continue;
 
-		ng_stats_ssrc_mos_entry_dict_avg(ent, "average MOS", &se->average_mos, se->stats_blocks.length);
+		ng_stats_ssrc_mos_entry_dict_avg(ent, "average MOS", &se->average_mos, (se->stats_blocks.length - se->no_mos_count));
```

Comment:
This may end up dividing by zero now - please add safeguards to protect from that.

Reply:
very true

Reply:
just pushed the safety and made normal condition tests